### DISCUSSION TITLE
Thank you for the AMAZING work!

### DISCUSSION THREAD

**Discussion by stdedos** [pfcao/abookmark#12] [General]
The extension is super-complicated ... but it has **SO**. **MANY**. **OPTIONS**.

I can literally do anything _(almost)_ I have dreamt of.

Thank you for your effort ❤️

**Comment by pfcao** [comment]
Thanks.

Maybe it is the most complicated one in chrome. I think it's mainly because it integrates a lot of features, which may be equivalent to five separate apps. The benefit of integration is that it can be interoperable, but it also increases the difficulty of getting started. 

It's a bit like sidebery of firefox, but the design ideas are different. Sidebery's main job is tab management, and it can also operate bookmarks. Abookmark is mainly responsible for bookmark management, and tabs are also mapped as bookmarks.  Abookmark is more abstract.
